wc_test.go

```diff
@@ -29,4 +30,19 @@ func TestWc(t *testing.T) {
 		assert.Equal(t, 0, wc.Words)
 		assert.Equal(t, 1, wc.Chars)
 	})
+
+	t.Run("Scanner error", func(t *testing.T) {
+		wc := (&Stream{
+			r: readerFn(func(_ []byte) (int, error) {
+				return 0, fmt.Errorf("oops")
+			}),
+		}).Wc()
+
+		assert.Equal(t, 0, wc.Lines)
+		assert.Equal(t, 0, wc.Words)
+		assert.Equal(t, 0, wc.Chars)
+
+		_, err := wc.ToString()
+		require.Error(t, err, "oops")
```

Comment:
Thanks for adding the test!
This function doesn't check for the error message: https://github.com/stretchr/testify/blob/78aedbf43344684e22e08f06b695eefff70fadaa/require/require.go#L289


You might want to use `require. ErrorContains` or `require.ErrorIs`, but I'm OK with just checking `require.Error(t, err)`. https://github.com/stretchr/testify/blob/78aedbf43344684e22e08f06b695eefff70fadaa/require/require.go#L328